CONTRIBUTING.md

```diff
@@ -96,52 +100,71 @@ Don't forget to unlink the package once you're done.
+- [Issues](../../issues)
+- [Discussions](../../discussions)
+- [Google group](https://groups.google.com/g/fabricjs)
+- [Stackoverflow](http://stackoverflow.com/questions/tagged/fabricjs)
+
+
+## 🚀 Pull Requests 
+
+Fabric is an open source project 🦄 and as such depends on the **genuine effort** of individuals and the community as a whole.
+**Join Us** to make Fabric better 🌺 . 
 
 ### Getting Started
 
-First of all, take a look at [fabric-gotchas](http://fabricjs.com/fabric-gotchas). It might save you time, effort and frustration.
+Read this section through and take a look at [**GOTCHAS**](./GOTCHAS.md).
 
 Follow [Developing](#-developing-) **OR** [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/) and start coding!
 
-### ✅ Guidelines
 
-- **Code style & notes:** Make sure you have complied with the [guidelines](https://github.com/fabricjs/fabric.js/wiki/How-to-contribute-to-Fabric#code-style-notes)
-- **⛔ Distribution files:** Do your changes only in the [source files](https://github.com/fabricjs/fabric.js/tree/master/src). Don't include the [distribution files](https://github.com/fabricjs/fabric.js/tree/master/dist) in your commit.
-- **Add tests**: Tests are vital, invest time to extend the them, see [TESTING](#🧪-testing).
-- **Add documentation:** Fabric uses [JSDoc 3] for documentation. The generated documentation can be found at [fabricjs.com](http://fabricjs.com/docs), see [DOCS](#🔎-improving-docs).
-- **One pull request per feature/bug**. Create a new branch for every pull request. If you want to do more than one thing, send multiple pull requests.
-- **And there you go!** If you still have questions we're always happy to help. Also feel free to consult [wiki](https://github.com/fabricjs/fabric.js/wiki/How-to-contribute-to-Fabric).
+### ✅ Guidelines 
 
-## 🧪 Testing
+- **Code Style** \
+Fabric uses [`prettier`](https://prettier.io/) to format files and [`eslint`](https://eslint.org/) for linting (`npm run lint -- --fix`).\
+To enjoy a seamless dev experience add the [`Prettier - Code formatter`](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension via the extensions toolbar in VSCode.
+- **⛔ `dist`** \
+Commit changes to [source files](https://github.com/fabricjs/fabric.js/tree/master/src). Don't commit the generated [distribution files](https://github.com/fabricjs/fabric.js/tree/master/dist).
+- **Tests** \
+PRs must be backed with relevant tests, follow [TESTING](#-testing).
+- **Docs** \
+Add relevant comments to your code if necessary using [JSDoc 3](https://jsdoc.app/) and update relevant guides.\
+The generated documentation can be found at [fabricjs.com](http://fabricjs.com/docs), see [DOCS](#🔎-improving-docs).
+- **Changelog**\
+Add a consice listing to the [**CHANGELOG**](./CHANGELOG.md) describing what has changed.
```

Comment:
typo: concise

Reply:
done
I wonder if there's a vscode plugin for typos in md files

Reply:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
![image](https://user-images.githubusercontent.com/34343793/190994505-a4085bb1-7058-4361-8a2e-d5362dfe9540.png)

good stuff